app/models/solidus_stripe/payment_method.rb

```diff
@@ -124,11 +124,12 @@ def find_intent_for_order(order)
       end
 
       def find_intent_for(payment)
```

Comment:
nit, missing "is missing" in the commit message?

Reply:
Thanks! Fixed